src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/jobs/adhoc/createtasks/JsonResourceCaseRetrieverService.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.wataskmonitor.services.jobs.adhoc.createtasks;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.wataskmonitor.domain.jobs.RetrieveCaseListParam;
+import uk.gov.hmcts.reform.wataskmonitor.domain.jobs.adhoc.createtasks.JsonResourceCaseList;
+import uk.gov.hmcts.reform.wataskmonitor.services.jobs.ResourceEnum;
+import uk.gov.hmcts.reform.wataskmonitor.services.jobs.RetrieveCaseListService;
+import uk.gov.hmcts.reform.wataskmonitor.utils.ObjectMapperUtility;
+import uk.gov.hmcts.reform.wataskmonitor.utils.ResourceUtility;
+
+@Component
+public class JsonResourceCaseRetrieverService implements RetrieveCaseListService<RetrieveCaseListParam> {
```

Comment:
JsonResourceCaseRetrieverService - can we add a unit test for this

Reply:
this is covered here ObjectMapperUtilityTest

Reply:
done